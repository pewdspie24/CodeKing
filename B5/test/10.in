38
2 25
2 24
1 28
3 23
2 29
1 25
3 26
3 26
2 26
2 23
3 29
1 30
1 20
3 30
3 23
1 29
3 30
2 21
2 25
1 24
1 23
1 28
3 26
1 21
1 30
3 22
3 29
2 29
1 30
1 26
2 29
1 26
2 24
2 24
1 23
2 27
1 28
1 26
