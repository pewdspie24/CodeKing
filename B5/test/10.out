288230376151711744
536870912
67108864
1162261467
10460353203
72057594037927936
524288
281474976710656
274877906944
4398046511104
31381059609
281474976710656
1125899906842624
2097152
1099511627776
7625597484987
1162261467
4398046511104
2097152
282429536481
4194304
18014398509481984
1048576
94143178827
281474976710656
1125899906842624
16777216
22876792454961
